GoogleForCreators/web-stories-wp
# Complete Text Sets Panel

## Summary
Uses the pieces required for the panel added in previous PR-s and puts these together.


## Relevant Technical Choices



## To-do

- [x] Unit tests (Karma to be implemented within #4514 



## User-facing changes
N/A - behind the feature flag


## Testing Instructions
- Open the Text Sets panel and compare it with[ Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=837%3A1962) to see if it looks as expected
- Verify the filtering works as expected


---



Fixes #4080
